src/common/http/platform/beast/http.cpp

```diff
@@ -458,6 +458,7 @@ static inline error::Error AddProxyAuthHeader(OutgoingRequest &req, BrokenDownUr
 		return ex_encoded_creds.error();
 	}
 	req.SetHeader("Proxy-Authorization", "Basic " + ex_encoded_creds.value());
+	log::Warning("Avoid Basic Auth if possible, and make sure if it's used, it's through HTTPS");
```

Comment:
I suggest we use the exact same terminology than in [Wikipedia](https://en.wikipedia.org/wiki/Basic_access_authentication):

```suggestion
	log::Warning("Avoid Basic access authentication if possible, and make sure if it's used, it's through HTTPS");
```

Or, at least, let's please spell out the full world
```suggestion
	log::Warning("Avoid Basic Authorization if possible, and make sure if it's used, it's through HTTPS");
```

Reply:
@lluiscampos I agree, but basic auth means basic authentication, not authorization. (The fact that there is `Authorization` in the HTTP header is annoying / confusing).

Also, I don't think we need to capitalize this.

See:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication

```suggestion
	log::Warning("Avoid using basic authentication if possible, and make sure if it's used, it's through HTTPS");
```

Reply:
> The fact that there is Authorization in the HTTP header is annoying / confusing

Yeah, I wrote it as an alternative to match 1:1 the HTTP header. I still prefer my first suggestion (and with no capitalization, as suggested on top of it)